release/release_gold.sh

```diff
@@ -183,11 +183,14 @@ function prepare_next_release_branch {
 			-e "s/release.info.version.display.name\[release-private\]=.*/release.info.version.display.name[release-private]=${product_group_version^^}.${next_project_version_suffix}/" \
 			"${_PROJECTS_DIR}/liferay-portal-ee/release.properties"
 
-		commit_to_branch_and_send_pull_request \
-			"${_PROJECTS_DIR}/liferay-portal-ee/release.properties" \
-			"Prepare ${quarterly_release_branch_name}" \
-			"${quarterly_release_branch_name}" \
-			"Prep next"
+		if [[ ! " ${@} " =~ " --test " ]]
+		then
+			commit_to_branch_and_send_pull_request \
+				"${_PROJECTS_DIR}/liferay-portal-ee/release.properties" \
+				"Prepare ${quarterly_release_branch_name}" \
+				"${quarterly_release_branch_name}" \
+				"Prep next"
+		fi
```

Comment:
@kiwm, shouldn't the block `if...else` below be included inside this `if` which is only executed when not running the test `test_release_gold.sh`?

From what I could understand, the `if...else` below should only be executed after the execution of the function `commit_to_branch_and_send_pull_request`, is this correct?